features/math.yml

```diff
@@ -0,0 +1,51 @@
+name: Math
+description: Static properties and methods for mathematical constants and functions.
+spec: https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-math-object
+status:
+  compute_from: javascript.builtins.Math
+compat_features:
+  - javascript.builtins.Math
+  - javascript.builtins.Math.E
+  - javascript.builtins.Math.LN10
+  - javascript.builtins.Math.LN2
+  - javascript.builtins.Math.LOG10E
+  - javascript.builtins.Math.LOG2E
+  - javascript.builtins.Math.PI
+  - javascript.builtins.Math.SQRT1_2
+  - javascript.builtins.Math.SQRT2
+  - javascript.builtins.Math.abs
+  - javascript.builtins.Math.acos
+  - javascript.builtins.Math.acosh
+  - javascript.builtins.Math.asin
+  - javascript.builtins.Math.asinh
+  - javascript.builtins.Math.atan
+  - javascript.builtins.Math.atan2
+  - javascript.builtins.Math.atanh
+  - javascript.builtins.Math.cbrt
+  - javascript.builtins.Math.ceil
+  - javascript.builtins.Math.clz32
+  - javascript.builtins.Math.cos
+  - javascript.builtins.Math.cosh
+  - javascript.builtins.Math.exp
+  - javascript.builtins.Math.expm1
+  - javascript.builtins.Math.f16round
```

Comment:
Note - this has no actual support across any browser, and is only experimental in FF